uclaacm/anti-resume-project
# feat: API for writing data to google sheets

## Summary
Adds an endpoint for writing data to the anti-resume spreadsheet. Uses a service account and spreadsheets.values.append() to write to the spreadsheet.
Endpoint is http://localhost:3000/api/addResume 
No dependencies

## Test Plan
1. yarn dev
2. open Postman
3. make a Post request to http://localhost:3000/api/addResume with body
```
{
   "resume" : 
   {
     "user": "mattchoo",
    "iHave": "L1\nL2\nL3",
    "toInforms": "LL1\nLL2\nLL3",
    "image": "http://www.imagelink.com/",
    "year": "2022"
  }
}
```
4. see if data is added to https://docs.google.com/spreadsheets/d/1v430W5Ap-1f6Xi_N61VafYH12exNM-qEqra12RCU9pc/edit?usp=sharing
